code/game/jobs/job_controller.dm

```diff
@@ -673,6 +673,7 @@ var/global/datum/controller/occupations/job_master
 	var/datum/spawnpoint/spawnpos
 	var/fail_deadly = FALSE
 	var/obj/belly/vore_spawn_gut
+	var/absorb_choice = "No" //CHOMPAdd - Ability to start absorbed with vorespawn
```

Comment:
Please don't use strings for numeric choices! Display the texts UI side!

use defines to set names in code.
#define SPAWN_ABSORBED 1


In this case, 3 Flags would even be preferred, flag Math exists on the soulcatcher to show the implementation for the UI with multiple states.